dbt/model.py

```diff
@@ -68,7 +68,12 @@ def config(self):
            - active project config
            - in-model config
         """
+
         defaults = {"enabled": True, "materialized": "view"}
+
+        if self.node_type == NodeType.Seed:
+            defaults['materialized'] = 'seed'
```

Comment:
@cmcarthur I really like this approach because users can make a new materialization, then specify it in the `seeds:` part of their `dbt_project.yml` file. Woooo

Reply:
wooooooooo